views/admin/admin-page.php

```diff
@@ -240,7 +241,103 @@
+					</div>
+				</div><?php // three-feature ?>
+
+				<header class="secondary-header j-int">
+					<h2 title="<?php esc_attr_e( 'Track your growth', 'jetpack' ); ?>"><?php esc_html_e( 'Track your growth', 'jetpack' ); ?></h2>
+				</header>
+
+				<div class="j-feature-img">
+					<img srcset="<?php echo plugins_url( 'images/connection-landing/stats-example-sm.png', JETPACK__PLUGIN_FILE ); ?> 1x,
+					<?php echo plugins_url( 'images/connection-landing/stats-example-med.png', JETPACK__PLUGIN_FILE ); ?> 1.5x,
+					<?php echo plugins_url( 'images/connection-landing/stats-example-lrg.png', JETPACK__PLUGIN_FILE ); ?> 2x" src="<?php echo plugins_url( 'images/connection-landing/stats-example-med.png', JETPACK__PLUGIN_FILE ); ?>" alt="<?php esc_attr( 'View detailed insights and analytics about your site with Jetpack', 'jetpack' ); ?>" />
+				</div>
+
+			</div><?php // jp-card ?>
+
+
+			<div class="j-security feature-container jp-card">
+				<header class="first-header j-int">
+					<h2 title="<?php esc_attr_e( 'Site Security and Peace of Mind', 'jetpack' ); ?>"><?php esc_html_e( 'Site Security and Peace of Mind', 'jetpack' ); ?></h2>
+				</header>
+
+				<div class="three-feature j-row">
+					<div class="j-col j-sm-12 j-md-12 j-lrg-4">
+						<h3 title="<?php esc_attr_e( 'Block site attacks', 'jetpack' ); ?>"><?php esc_html_e( 'Block site attacks', 'jetpack' ); ?></h3>
+						<p><?php esc_html_e( 'Gain peace of mind with Protect, the tool that has blocked billions of login attacks on millions of sites.', 'jetpack' ); ?></p>
+					</div>
+					<div class="j-col j-sm-12 j-md-12 j-lrg-4">
+						<h3 title="<?php esc_attr_e( 'Live site monitoring​', 'jetpack' ); ?>"><?php esc_html_e( 'Live site monitoring​', 'jetpack' ); ?></h3>
+						<p><?php esc_html_e( 'Stress less. Monitor will send you real-time alerts if your site ever goes down.', 'jetpack' ); ?></p>
+					</div>
+					<div class="j-col j-sm-12 j-md-12 j-lrg-4">
+						<h3 title="<?php esc_attr_e( 'Automatic Updates', 'jetpack' ); ?>"><?php esc_html_e( 'Automatic updates', 'jetpack' ); ?></h3>
+						<p><?php esc_html_e( 'With Manage, you’ll never be behind on a security release or waste time updating multiple sites.', 'jetpack' ); ?></p>
+					</div>
+				</div><?php // three-feature ?>
+			</div><?php // jp-card ?>
+
+			<div class="j-security feature-container jp-card">
+				<header class="secondary-header j-int">
+					<h2 title="<?php esc_attr_e( 'Optimized Images, Lightning Fast', 'jetpack' ); ?>"><?php esc_html_e( 'Optimized Images, Lightning Fast', 'jetpack' ); ?></h2>
+					<p><?php esc_html_e( 'Jetpack utilizes the state-of-the-art WordPress.com content delivery network to load your gorgeous images super fast optimized for any device, and it’s completely free.', 'jetpack' ); ?></p>
+				</header>
+
+				<div class="j-feature-img">
+					<img srcset="<?php echo plugins_url( 'images/connection-landing/feature-photon-sm.jpg', JETPACK__PLUGIN_FILE ); ?> 1x,
```

Comment:
How about using Photon to serve those images? It would be relevant here :)

You can use `jetpack_photon_url()` to use Photon here. It accepts 2 parameters: the image URL, and optional image parameters (size, filters, [...](https://developer.wordpress.com/docs/photon/))